content/dev/dev-guide.md

```diff
@@ -2,66 +2,21 @@
 
 ## What is this guide?
 
-This is Tlon's guide for new developers that want to build applications on Urbit. The goal of this guide is to take you from a fresh machine to a working development environment for Urbit applications. This guide assumes macOS, but Linux is also supported (see the Urbit foundation docs for Linux support) It's important this guide remains clear and correct, if you find something broken please let us know at support@tlon.io
-
-## Pre-requisites
-
-### Homebrew
-
-- Homebrew is a package manager for macOS, we'll need it installed in order to install `git-lfs`.
-- ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
-
-### Git LFS
-
-- A package that enables git to work with large files, we'll install this via brew.
-- ```brew install git-lfs```
-
-### Visual Studio Code
-
- - While not technically a prerequisite, there's a hoon syntax highlighting plugin available and this is the editor we recommend [Visual Studio Code](https://code.visualstudio.com/).
- - For the small text file edits in this guide, we'll simply use `nano`.
-
-### The Urbit Git repository
-
-  - We're going to need a local copy of the Urbit source code.
-     - ```bash
-        git clone https://github.com/urbit/urbit ~/urbit-git
-       ```
-  - Pull down the files stored using git lfs. 
-    - ```bash
-      cd ~/urbit-git
-      git lfs install
-      git lfs pull
-      ```
-
-### The Urbit binary
-
- - First we need to create a directory for the urbit binary, we'll create this in your home directory.
-   - ```bash
-     mkdir ~/urbit-dev
-     ```
- - Next we're going to pull down the latest urbit binary from urbit.org from the newly created directory.
-    - ```bash
-      cd ~/urbit-dev
-      curl -JLO https://urbit.org/install/mac/latest
-      ```
-  - Now we'll untar the contents of the download into our newly created directory.
-    - ```bash 
-      tar -zxvf ./darwin.tgz --strip=1
-      ```
-  - With the urbit binary installed we're ready for the basic dev workflow.
+This is Tlon's guide for new developers that want to build applications on Urbit. The goal of this guide is to take you from a fresh machine to a working development environment. We assume basic knowledge of the command line and somewhat prioritize macOS, but Linux is fully supported (see the Urbit foundation docs for more details). It's important that this guide remains clear and correct, if you find something broken please let us know at support@tlon.io
 
+>  The following assumes you have an urbit binary installed and the git repo cloned. Follow the steps in External Pre-Requisites before continuing if you do not.
```

Comment:
I don't think we should make this assumption, even if they have the repo cloned they probably don't have git-lifs. IME it's better to just explicitly have this in order for now.